addons/forum-toolbar/addon.json

```diff
@@ -0,0 +1,150 @@
+{
+  "name": "Forum toolbar buttons",
+  "description": "Adds buttons in the forum editing toolbar to insert BBCode, such as [center] or [color].",
+  "tags": ["community", "forums"],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["editingScreens"]
+    }
+  ],
+  "userstyles": [
+    {
+      "url": "userstyle.css",
+      "matches": ["editingScreens"]
+    }
+  ],
+  "settings": [
+    {
+      "name": "Wikipedia",
```

Comment:
Instead of using a separate setting for each of these, could the "link to ..." options be items of a menu?